releasenotes/notes/CQM.is_linear-25156a452fc4e012.yaml

```diff
@@ -0,0 +1,3 @@
+---
+features:
+  - Add ``ConstrainedQuadraticModel.is_linear()`` method.
```

Comment:
Is this generated automatically by Reno?

Reply:
No, the file is hand written by me. Reno then turns the files in `releasenotes/` into a changelog. See [here](https://docs.ocean.dwavesys.com/en/stable/docs_dimod/release_notes.html) and [here](https://github.com/dwavesystems/dimod/releases). The former is generated as part of the sphinx build (see [here](https://github.com/dwavesystems/dimod/blob/main/docs/release_notes.rst)), the latter I just generate locally and then copy in.

Reply:
I should clarify that the file is generated by reno, but the contents are hand written.